docs/docs/indispensable.md

```diff
@@ -174,6 +174,11 @@ These interfaces essentially provide a mapping between custom types and low-leve
 This also means, a direct serialization of such custom types is valuable for debugging, but not for encoding.
 **Hence, decoding a kotlinx.serialization output of those classes is unsupported.**
 
+In addition, `CryptoPublicKey`, `CryptoPrivateKey`, `X509Certificate`, `Pkcs10CertificationRequest` also implement `PemEncodable`,
+while their respective companions implement `PemDecodable`.
+As the names suggest, this brings about `encodeToPem` and `decodeFromPem` functions doing what their names imply:
```

Comment:
"As the names suggest, this brings ... function doing what their names imply:" feels a tiny bit too smug and self-satisfied

Reply:
agreed, one too many